<a href="https://colab.research.google.com/github/Ryan456712/Python-Basics/blob/main/Math_Ex1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# MATH PYTHON EXERCISE 1

1. Import math correctly

In [10]:
import math

2. Convert 90 degrees to radians and print it

In [11]:
angle = 90
radians = math.radians(angle)
print(radians)

1.5707963267948966


3. Calculate sin 70, cos 70, tan 70

In [15]:
angle = 70
radians = math.radians(angle)
print(radians)

print("sin of 70 degrees:", math.sin(radians))

print("cos of 70 degrees:", math.cos(radians))

print("tan of 70 degrees:", math.tan(radians))

1.2217304763960306
sin of 70 degrees: 0.9396926207859083
cos of 70 degrees: 0.3420201433256688
tan of 70 degrees: 2.7474774194546216


4. Calculate sin inverse (0.9396926207859083) cos inverse (0.342020143325668) and tan inverse (2.747477419454621)

In [18]:
print("arcsin of  degrees:", (math.asin(0.9396926207859083)))
print("arccos of 70 degrees:", (math.acos(0.3420201433256688)))
print("arctan of 70 degrees:", (math.atan(2.7474774194546216)))

arcsin of 70 degrees: 1.2217304763960306
arccos of 70 degrees: 1.2217304763960306
arctan of 70 degrees: 1.2217304763960306


5. Calculat the area of a circle with radius 7

In [72]:
r = 7
area = math.pi * (math.pow(r,2))
print(area)

153.93804002589985


6.Calculate the circumference of a circle with radius 7

In [79]:
 r = 7

def calculate_circumference(radius):
    circumference = (2 * math.pi * r)
print(circumference)

43.982297150257104


7. Find the cube root of 343 using the math.pow() function

In [53]:

def cube_root(number):
    return math.pow(number, 1/3)

number = 343
result = cube_root(number)
print(f"The cube root of {number} is {result}")


The cube root of 343 is 6.999999999999999


8. Calculate the floor and ceil of 7.83.

In [58]:
print("math.floor(7.83) : ", math.floor(7.83))
print("math.ceil(7.83) : ", math.ceil(7.83))

math.floor(7.83) :  7
math.ceil(7.83) :  8


9. Verify identity of (sin45)^2 + (cos45)^2 is approximately equal to 1

In [71]:
angle = 45
radians = math.radians(angle)
print(radians)

print("sin of 70 degrees:", math.sin(radians))

print("cos of 70 degrees:", math.cos(radians))

angle_rad = math.radians(45)

math.sin(angle_rad)**2 + math.cos(angle_rad)**2

0.7853981633974483
sin of 70 degrees: 0.7071067811865475
cos of 70 degrees: 0.7071067811865476


1.0

10. An object is dropped from a height of 78.4 meters.
How long will it take to hit the ground?
(Use the formula: t = √2h/g , g = 9.8

11. Area of an Equilateral Triangle
Given the side length of an equilateral triangle is 6 units, calculate its area.
Use the formula: Area = √(3/4) * (side)^2